recipes/eastl/all/conanfile.py

```diff
@@ -50,67 +53,90 @@ def _minimum_compilers_version(self):
         }
 
     def export_sources(self):
-        self.copy("CMakeLists.txt")
-        for patch in self.conan_data.get("patches", {}).get(self.version, []):
-            self.copy(patch["patch_file"])
+        export_conandata_patches(self)
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("eabase/2.09.06")
+        self.requires("eabase/2.09.12", transitive_headers=True)
```

Comment:
Okay, we will have the follow error in case is not exposed:

```
p/include/EASTL/internal/config.h:61:11: fatal error: 'EABase/eabase.h' file not found
        #include <EABase/eabase.h>
```